src/main/java/com/puppycrawl/tools/checkstyle/DetailNodeTreeStringPrinter.java

```diff
@@ -170,37 +167,8 @@ private static String getIndentation(DetailNode node) {
      */
     private static DetailNode parseFile(File file) throws IOException {
         final FileText text = new FileText(file.getAbsoluteFile(),
-            System.getProperty("file.encoding", StandardCharsets.UTF_8.name()));
+                System.getProperty("file.encoding", StandardCharsets.UTF_8.name()));
```

Comment:
One more indentation change to be removed

Reply:
fixed